CCPAPI/PAPI/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+##########################################################################
+#                                                                        #
+#                               PAPI                                     #
+#                                                                        #
+#  This program is free software; you can redistribute it and/or modify  #
+#  it under the terms of the GNU Library General Public License as       #
+#  published by the Free Software Foundation; version 2 or later of the  #
+#  License.                                                              #
+#                                                                        #
+#  This program is distributed in the hope that it will be useful,       #
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+#  GNU General Public License for more details.                          #
+#                                                                        #
+#          Copyright 2020 Paul RASCLE www.openfields.fr                  #
+#                                                                        #
+##########################################################################
+
+message(STATUS "Python API")
```

Comment:
Question about this actually:

When you are running cmake are all the things related to this project coming out at the same time together?

What I want to prevent is a bunch of status stuff being output interspersed with the rest of the build. It gets noisy. If everything from this project is output as one block of info then ignore the above :-)